### PR TITLE
solution

### PR DESCRIPTION
**Pull request by kq333**

### CODE REVIEW

**Review by SuaBy514 — APPROVED** [review]

**Review by michalcywka — CHANGES_REQUESTED** [review]
We don't want magic numbers here :) Checklist point 1 and 4

**Review by michalcywka — APPROVED** [review]

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const mdDiscount = 20;
+  const lgDiscount = 50;
+  const price = 40;
+
+  if (days <= 2) {
+    return days * price;
```
Comment (michalcywka) [inline src/calculateRentalCost.js:36]: `days * price` is repeated, can me move it to a variable? :)

**Review by michalcywka — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,22 @@
...
+
+  if (days <= 2) {
+    return priceBeforDiscount;
+  }
+
+  if (days >= 3 && days <= 6) {
+    return priceBeforDiscount - mdDiscount;
+  }
+
+  if (days >= 7) {
```
Comment (michalcywka) [inline src/calculateRentalCost.js:44]: Do we really need this if here?

**Review by michalcywka — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 2 above)
Comment (d-wasilewski) [reply src/calculateRentalCost.js:44]: we don't. It's checklist point 3

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

**Pull request closed without merging**